dev/conductor/core/lib/src/start.dart

```diff
@@ -257,10 +256,18 @@ class StartContext {
   /// If validations should be overridden.
   final bool force;
 
+  late final EngineRepository engine;
+  late final FrameworkRepository framework;
+
+  /// Should be called after [run] to obtain the engine checkout directory.
+  Future<Directory> get engineCheckoutDirectory => engine.checkoutDirectory;
```

Comment:
I don't think these getters are necessary, since `engine` and `framework` are public (callers can call `engine.checkoutDirectory` themselves)

Reply:
Is it possible to call `engine.checkoutDirectory` on the engine or framework repositories of a `startContext` class in that case?